File: MY PROGRAMS/LB/JAVA/NUMBER/a8q3.java
/*	3.Write a program to find factorial of given number.

	Input:	5!
	Output:	120

	Input:	-5!
	Output:	120

	Input:	4!
	Output:	24	*/




import java.util.*;

class Assignment8
{
	public void Display(int ino)
	{
		int i=0,ifact=1;

		if(ino<0)
		{
			ino=-ino;
		}

		for(i=ino;i>0;i--)
		{
			ifact=ifact*i;
		}

		System.out.println("Factorial of the given no. is "+ifact);
	}
}

class a8q3
{
	public static void main(String arg[])
	{
		Scanner sobj=new Scanner(System.in);

		int ivalue=0;

		System.out.println("Enter the no.: ");
		ivalue=sobj.nextInt();

		Assignment8 aobj=new Assignment8();

		aobj.Display(ivalue);
	}
}

File: MY PROGRAMS/LOGIC BUILDING-13 ASSIGNMENTS/a8q3.c
/* Write a program to find factorial of given number.

Input:5
Output:120

Input:-5
Output:120

Input:4
Output:24

*/




#include<stdio.h>

int Factorial(int ino)
{
	int ifact=1,i=0;

	if(ino<0)
	{
		ino=-ino;
	}

	for(i=1;i<=ino;i++)
	{
		ifact=ifact*i;
	}	

	return(ifact);
}

int main()
{
	int ivalue=0,iret=0;

	printf("Enter the no.:");
	scanf("%d",&ivalue);

	iret=Factorial(ivalue);

	printf("Factorial is %d",iret);

	return(0);
}
